src/main/flight/mixer.c

```diff
@@ -272,29 +272,22 @@ static void calculateThrottleAndCurrentMotorEndpoints(timeUs_t currentTimeUs)
 #define CRASH_FLIP_DEADBAND 20
 #define CRASH_FLIP_STICK_MINF 0.15f
 
-static void applyFlipOverAfterCrashModeToMotors(void)
+static void applyFlipOverAfterCrashModeToMotors(const float flipAngleModifier)
```

Comment:
```suggestion
static void applyFlipOverAfterCrashModeToMotors()
```
You can prevent the need to input `flipAngleModifier` by implementing the feature completely inside of `applyFlipOverAfterCrashModeToMotors()`. The turtle mode code is completely encapsulated in here. This would also clean up `mixTable()` again.

Reply:
Thanks very much for the coding changes you suggested via Discord.  I agree, it's much better to encapsulate the code properly.  Your suggestions demonstrate my mediocre coding ability rather nicely.  They are implemented now.  Thank you!